README.md

```diff
@@ -12,219 +12,249 @@
 
-Siamese Recurrent Architectures for Learning Sentence Similarity.
-[website](https://dl.acm.org/citation.cfm?id=3016291) | [paper](http://www.mit.edu/~jonasm/info/MuellerThyagarajan_AAAI16.pdf)
+   Match-SRNN: Modeling the Recursive Matching Structure with Spatial RNN. *IJCAI 2016*.
+   [website](https://arxiv.org/abs/1604.04378) | [paper](https://arxiv.org/pdf/1604.04378.pdf)
 
-Learning Text Similarity with Siamese Recurrent Networks.
-[website](https://sites.google.com/site/repl4nlp2016/accepted-papers) | [paper](http://www.aclweb.org/anthology/W16-16#page=162)
+6. **MultiGranCNN** [[code]]() [[pdf]](https://aclanthology.info/pdf/P/P15/P15-1007.pdf) ![status](artworks/not-in-plan.svg)
 
+   MultiGranCNN: An Architecture for General Matching of Text Chunks on Multiple Levels of Granularity. *ACL 2015*.
+   [website](https://arxiv.org/abs/1604.04378) | [paper](https://aclanthology.info/pdf/P/P15/P15-1007.pdf)
 
-#### ESIM
-![status](artworks/not-in-plan.svg)
+7. **ABCNN** [[code]]() [[pdf]](https://arxiv.org/pdf/1512.05193.pdf) ![status](artworks/not-in-plan.svg)
 
-Enhanced LSTM for Natural Language Inference * ACL 2017
-[website](https://arxiv.org/abs/1609.06038) | [paper](https://arxiv.org/pdf/1609.06038.pdf)
+   ABCNN: Attention-Based Convolutional Neural Network for Modeling Sentence Pairs. *ACL 2016*.
+   [website](https://arxiv.org/abs/1512.05193) | [paper](https://arxiv.org/pdf/1512.05193)
 
-#### MCAN
-![status](artworks/not-in-plan.svg)
+8. **MwAN** [[code]]() [[pdf]](https://www.ijcai.org/proceedings/2018/0613.pdf) ![status](artworks/progress.svg)
 
-Multi-Cast Attention Networks for Retrieval-based Question Answering and Response Prediction
-[website](https://arxiv.org/abs/1806.00778) | [paper](https://arxiv.org/pdf/1806.00778.pdf)
+   Multiway Attention Networks for Modeling Sentences Pairs. *IJCAI 2018*.
 
-#### MIX
-![status](artworks/not-in-plan.svg)
+   [website](https://www.ijcai.org/proceedings/2018/0613.pdf) | [paper](https://www.ijcai.org/proceedings/2018/0613.pdf)
 
-MIX: Multi-Channel Information Crossing for Text Matching
-[website](http://www.kdd.org/kdd2018/accepted-papers/view/mix-multi-channel-information-crossing-for-text-matching) | [paper](https://sites.ualberta.ca/~dniu/Homepage/Publications_files/hchen-kdd18.pdf)
+9. **…**
 
-#### HiNT
-![status](artworks/not-in-plan.svg)
+### Community Question Answer
 
-Modeling Diverse Relevance Patterns in Ad-hoc Retrieval
-[website](https://arxiv.org/abs/1805.05737) | [paper](https://arxiv.org/pdf/1805.05737.pdf)
+---
+
+**Community Question Answer** is to automatically search for relevant answers among many responses provided for a given question (Answer Selection), and search for relevant questions to reuse their existing answers (Question Retrieval).
+
+Some benchmark datasets are listed in the following,
+
```

Comment:
We can add a passage as follows for TRECQA data here:

[**TRECQA**] TREC QA data set is created by  [Wang et. al.](http://www.aclweb.org/anthology/D07-1003)   from TREC QA track 8-13 data, with candidate answers automatically selected from each question’s document pool using a combination of overlapping non-stop word counts and pattern matching. This data set is one of the most widely used benchmarks for [answer sentence selection](https://aclweb.org/aclwiki/Question_Answering_(State_of_the_art)).